pssh/ssh_client.py

```diff
@@ -107,7 +115,7 @@ def __init__(self, host,
         self.user = user
         self.password = password
         self.pkey = pkey
-        self.port = port if port else 22
+        self.port = port
```

Comment:
Better written in one go like:

```
self.port = port if port else int(host_config['port']) \
    if 'port' in host_config else 22
```

Please also keep max line length to 80 for PEP-8 compliance.